core/App/components/modals/AlertModal.tsx

```diff
@@ -27,13 +29,13 @@ const AlertModal: React.FC<AlertModalProps> = ({ title, message, submit }) => {
   return (
     <Modal visible={true} transparent={true}>
       <SafeAreaView style={[styles.container]}>
-        <InfoBox
+        <PopupModal
```

Comment:
Maybe I'm wrong here but in my mind the polyp modal encapsulates `Modal` and `SafeAreaView` so it's a completely reusable component and we can removed the individually named modals.

Reply:
you're right, I've remove the `SafeAreaView` and the `Modal`

Reply:
Will they still be full screen?

Reply:
We probably want to wrap the `PopupModal` component with a `SafeAreaView`

Reply:
it seems to work fine without the `SafeAreaView`. The Popup is still full screen

Reply:
it was still full screen but I wrapped it in a `SafeAreaView` now